security/cmd/istio_ca/main.go

```diff
@@ -149,9 +156,7 @@ func runCA() {
 	}
 
 	glog.Info("Istio CA has started")
-
-	<-stopCh
-	glog.Warning("Istio CA has stopped")
+	select {} // wait forever
```

Comment:
Just curious why this change is needed?

Reply:
I felt it's better to change it because the last line is unreachable. `stopCh` passed to cache controller is actually expected to be called by `main()`, not the controller. So `<-stopCh` here causes main to block forever, but I think it's clearer to just use `select {}` here to prevent future confusion about the `stopCh` usage. See https://github.com/kubernetes/client-go/blob/master/tools/cache/controller.go#L97:8